changes/bug32315

```diff
@@ -0,0 +1,4 @@
+  o Major bugfixes (networking):
```

Comment:
We'll probable end up categorising this as minor when we do the changelog, because no-one noticed it was broken for a few years.
```suggestion
  o Minor bugfixes (networking):
```